{"links": {"self": {"href": "data/repositories/osrf/subt/issues/245/comments/54279119.json"}, "html": {"href": "#!/osrf/subt/issues/245#comment-54279119"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/245.json"}}, "type": "issue", "id": 245, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "Robot-to-robot communication problem"}, "content": {"raw": "Also, I verified `SubtRosRelay::OnMessage` was getting called correctly on the bridge. It appears to be the `ros::service::call(_req.dst_address(), req, res);` call that wasn\u2019t going through to the solution container correctly \\(until I manually called the service\\).", "markup": "markdown", "html": "<p>Also, I verified <code>SubtRosRelay::OnMessage</code> was getting called correctly on the bridge. It appears to be the <code>ros::service::call(_req.dst_address(), req, res);</code> call that wasn\u2019t going through to the solution container correctly (until I manually called the service).</p>", "type": "rendered"}, "created_on": "2019-10-09T17:58:43.606841+00:00", "user": {"display_name": "Malcolm Stagg", "uuid": "{eee13832-fdd5-4196-aa05-6cfdb1118c65}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beee13832-fdd5-4196-aa05-6cfdb1118c65%7D"}, "html": {"href": "https://bitbucket.org/%7Beee13832-fdd5-4196-aa05-6cfdb1118c65%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/8c08ab97dc600bddad36e4a58f8cc5afd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsMS-3.png"}}, "nickname": "malcolmst7", "type": "user", "account_id": "557058:08ef8b07-1500-4917-989c-89f91ce4beac"}, "updated_on": null, "type": "issue_comment", "id": 54279119}